dashboard/apps/prototype/models.py

```diff
@@ -141,13 +152,17 @@ class Project(models.Model):
     project_manager = models.ForeignKey(
         'Person', related_name='projects', null=True)
     client = models.ForeignKey('Client', related_name='projects', null=True)
-    discovery_date = models.DateField(null=True)
-    alpha_date = models.DateField(null=True)
-    beta_date = models.DateField(null=True)
-    live_date = models.DateField(null=True)
-    end_date = models.DateField(null=True)
+    discovery_date = models.DateField(null=True, blank=True)
+    alpha_date = models.DateField(null=True, blank=True)
+    beta_date = models.DateField(null=True, blank=True)
+    live_date = models.DateField(null=True, blank=True)
+    end_date = models.DateField(null=True, blank=True)
+    visible = models.BooleanField(default=True)
     raw_data = JSONField(null=True)
 
+    objects = ProjectManager()
```

Comment:
whist this works, i think it not explicit enough, therefore it could be difficult to remember and easy to make mistakes. might be better to do this instead:

```
 class ProjectManager(models.Manager):
     use_for_related_fields = True
 
     def visible_only(self):
         return self.filter(visible=True)
```

when used:

```
all_projects = Project.objects.all()
projects = Project.objects.visible_only()
```